accepted/diagnostics/source-link.md

```diff
@@ -103,7 +103,7 @@ Using this URL scheme, it is possible to generate Source Link for VSTS git repos
 {
     "$schema": "http://json-schema.org/draft-04/schema#",
     "title": "SourceLink",
-    "description": "A mapping of source file paths to URL's",
+    "description": "A mapping of source file paths to URLs",
```

Comment:
(Apostrophes are only used to pluralize acronyms when each letter of the acronym is followed by a dot. For acronyms without dots, `'s` is used for possessive.)